libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/Imp/Ratify.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Cardano.Ledger.Conformance.Imp.Ratify (spec) where
```

Comment:
I think it's better to handle this in a separate PR, because it entails moving all `Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.*` modules to the `testlib`. I think it'd be a good idea to do that anyways, but I'd rather not do it in this PR.

Reply:
Sorry, but this doesn't make sense.

Nothing needs to change, except location where this module lives and where it is listed in the cabal file. Some or all of these modules need to be made public too: https://github.com/IntersectMBO/cardano-ledger/blob/1467956ee0be00d9c32412d3994e3204888d1819/libs/cardano-ledger-conformance/cardano-ledger-conformance.cabal#L31-L49

`cardano-ledger-conformance` is not a package that we need to release nor will it ever be used as a library. So, there is really little reason to hide those modules.

So, please move this file into the test suite. This PR is not gonna get much bigger because of that.

Reply:
Currently this package only has a test suite and a library, are you talking about moving this to the test suite or should I make a `testlib` and then move the ImpTests there?

Reply:
To the test suite. This `spec` makes no sense as a library, since it is not reusable.
We don't need a `testlib` in this package